lib/v2/tingshuitz/shenzhen.js

```diff
@@ -0,0 +1,32 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const sortCreatedOnTime = (a, b) => {
+    const t1 = timezone(parseDate(a.createdOn, 'YYYY-MM-DD HH:mm:ss'), +8);
+    const t2 = timezone(parseDate(b.createdOn, 'YYYY-MM-DD HH:mm:ss'), +8);
+    if (t1 > t2) { return 1; };
+    if (t1 === t2) { return 0; };
+    if (t1 < t2) { return -1; };
+};
```

Comment:
No need to sort by reverse chronological again since RSSHub will do it.
https://github.com/DIYgod/RSSHub/blob/cbca59f3902416e81a7ca84a0b14aa0a6d54848f/lib/middleware/parameter.js#L56-L59